src/status_im2/contexts/share/events.cljs

```diff
@@ -4,12 +4,22 @@
             [quo2.foundations.colors :as colors]))
 
 (rf/defn copy-text-and-show-toast
+  "This function will copy the text to a clipboard maintained by re-frame
```

Comment:
Hey @siddarthkay, I think there's a problem with the existing approach that we both missed in our conversation.

Let's imagine the following:

1. The user copies the emoji hash in status-mobile. The user correctly sees the toast. Pressing again, nothing is showed, perfect.
2. Then the user switches to another app and copy something else there, thus replacing what's currently in the global clipboard.
3. The user goes back to Status and presses copy again, but no toast will be displayed and the text won't be copied.

So I think we should revisit this PR's solution.

Reply:
Hi @ilmotta : Thanks for thinking through this one. I knew something like this would happen and that is why the clipboard I'm maintaining in re-frame is separate from the global clipboard on the app  😉 
So we probably do not need to worry about this case 👍🏻

Reply:
I tested this exact flow, and it clearly seems to be a bug from the user's perspective @siddarthkay.

[untitled.webm](https://github.com/status-im/status-mobile/assets/46027/1ad5758c-97ac-46a8-b6ed-fc6c166ce622)

Reply:
ohh good catch! will look into it @ilmotta 
nice edge case you've discovered!

Reply:
@siddarthkay please, let me know when this one will be addressed. after that I will take PR into testing. thank you!

Reply:
@ilmotta : I got rid of the whole clipboard approach, the new approach is much neater :)

Reply:
@pavloburykh : this PR is ready for testing again.